package.json

```diff
@@ -5,6 +5,14 @@
   "author": "{{{ author }}}",
   "license": "ISC",
   "repository": "https://github.com/{{ owner }}/{{ repo }}.git",
+  "homepage": "https://probot.github.io/apps/{{ name }}/",
+  "bugs": "https://github.com/probot/{{ name }}/issues",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-plugin",
```

Comment:
we deprecated use of `probot-plugin`, so I'd prefer it not be included for any new apps

Reply:
I would remove it from the GitHub keywords, then. :)